src/main/java/ui/issuepanel/FilterPanel.java

```diff
@@ -125,7 +125,9 @@ private void setUpEventHandler() {
     private Node createFilterBox() {
         filterTextField = new FilterTextField("")
                 .setOnConfirm((text) -> {
+                    Platform.runLater(() -> ui.triggerEvent(new PanelReloadingEvent()));
                     applyStringFilter(text);
+                    Platform.runLater(() -> ui.triggerEvent(new PanelReloadedEvent()));
```

Comment:
It doesn't, currently, since it only shows the loading gif if the panel is loading issues from the repo currently selected in the RepositorySelector.

However, a simple change to remove that check should be able to produce this behaviour with some problems. The typical execution (locally) would be PanelReloadingEvent :arrow_right:  PanelReloadedEvent :arrow_right: RepoOpeningEvent :arrow_right: RepoOpenedEvent, which will cause the spinning gif to appear and disappear twice. However, it is possible that the RepoOpeningEvent is adding to the event queue before PanelReloadedEvent (due to asynchronous execution), which may cause the gif to end prematurely.

Would the above be sufficient for this PR, or it is preferable to solve the concurrency issue now?